test/fuzz/utility.h

```diff
@@ -11,9 +11,9 @@ namespace Envoy {
 namespace Fuzz {
 
 // Convert from test proto Headers to TestHeaderMapImpl.
-inline Http::TestHeaderMapImpl
-fromHeaders(const test::fuzz::Headers& headers,
-            const std::unordered_set<std::string>& ignore_headers = {}) {
+inline Http::TestHeaderMapImpl fromHeaders(
+    const test::fuzz::Headers& headers,
+    const std::unordered_set<std::string>& ignore_headers = std::unordered_set<std::string>({})) {
```

Comment:
LGTM. Is there a practical difference between `std::unordered_set<std::string>({})` and `std::unordered_set<std::string>()`?

Reply:
Probably not, will simplify.